3rdparty/python/constraints.txt

```diff
@@ -1,4 +1,4 @@
-# Generated by build-support/bin/generate_lockfile.sh on Mon Sep 28 09:49:00 PDT 2020
+# Generated by build-support/bin/generate_lockfile.sh on Sun Oct  4 22:30:50 PDT 2020
```

Comment:
You'll want to regenerate one more time now that you are importing `libcst`. The script to generate only considers requirements returned in `./pants dependencies --transitive`, and I suspect you ran this script before having had added the import to Python so it's not getting picked up here.

Reply:
Thank you! Done!